azurelinuxagent/common/osutil/ubuntu.py

```diff
@@ -50,7 +56,11 @@ def get_dhcp_pid(self):
         ret = shellutil.run_get_output("pidof dhclient3", chk_err=False)
         return ret[1] if ret[0] == 0 else None
 
-class UbuntuOSUtil(Ubuntu14OSUtil):
+
+class Ubuntu16OSUtil(Ubuntu14OSUtil):
```

Comment:
This will be either 17 or 18, I think. Convention has been to use LTS versions but I am ok either way.

Reply:
I took the approach that the latest version of Ubuntu is current, and if you need to make a change you use the version number.

 * UbuntuOSUtil is for >= 17.10
 * Ubuntu16OSUtil was created an now represents previous versions.  

This seems to be the style of the current code.